test/integration_test/migrs/migration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
```

Comment:
The specs should go under the specs directory. Also use a more appropriate name for the migration